modules/amf-registration/amf-registration-service/service.xml

```diff
@@ -29,7 +29,7 @@
 		<column name="accountName" type="String" />
 		<column name="emailAddress" type="String" />
 		<column name="gender" type="String" />
-		<column name="birthday" type="String" />
+		<column name="birthday" type="Date" />
```

Comment:
make sure to run `buildService` again so changes take effect in the service classes.